### PR TITLE
Pylint improvement fix unused-argument

### PR DESCRIPTION
**Pull request by jonathan-eq**

### CODE REVIEW

File: komodo/build.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def dfs(pkg, version, pkgs, repo):
     return flatten(dependencies)
 
 
-def rpm(pkg, ver, path, data, prefix, *args, **kwargs):
+def rpm(pkg, ver, path, prefix):
```
Comment (kwinkunks) [inline komodo/build.py:45]: Big chage, do you know for sure nothing is calling this function with positional args, or passing `*args` or `**kwargs`?

Comment (jonathan-eq) [reply komodo/build.py:45]: It should only be called from the make() function, but that is an assumption

Comment (kwinkunks) [reply komodo/build.py:45]: After chatting, I think I get it.

File: komodo/build.py [hunk 2]
```diff
@@ -64,20 +64,19 @@ def cmake(
     pkg,
     ver,
     path,
```
Comment (kwinkunks) [inline komodo/build.py:66]: Use `pkgpath` here to match other functions?

File: komodo/build.py (hunk 1 above)
Comment (kwinkunks) [inline komodo/build.py:45]: Use `pkgpath` here to match other functions?

File: komodo/build.py [hunk 3]
```diff
@@ -42,7 +42,7 @@ def dfs(pkg, version, pkgs, repo):
     return flatten(dependencies)
 
 
-def rpm(pkg, ver, path, data, prefix, *args, **kwargs):
+def rpm(pkg, ver, path, prefix):
     # cpio always outputs to cwd, can't be overriden with switches
```
Comment (kwinkunks) [inline komodo/build.py:46]: Recommend adding docstring to this and all other functions. Might save the next person a bit of time figuring this module out.

File: komodo/build.py [hunk 4]
```diff
@@ -313,30 +317,75 @@ def resolve(x):
...
-            hash=current.get("hash"),
-        )
+            makeopts = f"{makeopts} {extra_makeopts}"
+        makeopts = resolve(makeopts)
+
+        if make == "rpm":
+            rpm(pkg=package_name, ver=ver, path=pkgpath, prefix=prefix)
+        if make == "cmake":
+            cmake(
+                pkg=package_name,
```
Comment (kwinkunks) [inline komodo/build.py:329]: These could match too...

File: komodo/build.py (hunk 3 above)
Comment (jonathan-eq) [reply komodo/build.py:46]: I think that is a separate pylint issue, but it is definitely on the to-do list!

**Review by kwinkunks — APPROVED** [review]
Looks good Jonathan :)

**Pull request merged**